src/test/java/com/epam/atm/selenium/CreateNewCasePOTest.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.atm.selenium;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageobject.*;
+
+public class CreateNewCasePOTest extends BaseTest {
+    private static final String LOGIN = "yauhen_valodzin@epam.com.qa";
```

Comment:
as this always used in all tests move it to baseTest
private static final String LOGIN = "yauhen_valodzin@epam.com.qa";
private static final String PASSWORD = "Ceakt_1234";

Reply:
done